sktime/tests/test_softdeps.py

```diff
@@ -12,9 +12,11 @@
 import pkgutil
 import re
 from importlib import import_module
+from unittest.mock import patch
 
 import pytest
 
+from sktime.base._base import BaseObject
```

Comment:
could you move the import from `sktime.base` inside the test?

The base classes import the `utils`, we need to avoid circular imports.

Reply:
Solved